bigbluebutton-client/resources/prod/lib/bbb_webrtc_bridge_sip.js

```diff
@@ -564,6 +571,22 @@ function webrtc_hangup(callback) {
 	}
 }
 
+function releaseUserMedia() {
+	if (!!userMicMedia) {
+		console.log("Releasing media tracks");
+	
+		userMicMedia.getAudioTracks().forEach(function(track) {
+            track.stop();
+        });
+
+        userMicMedia.getVideoTracks().forEach(function(track) {
```

Comment:
I would suggest either to bring all the `userMicMedia.*` calls inside the `if` block
OR
to start the function body with `if (userMicMedia == null) { return; }`

Reply:
Never mind. I misread it due to the mixed indentation.